acceptance_tests/features/steps/view_collection_exercise_details.py

```diff
@@ -0,0 +1,21 @@
+from behave import given, when, then
+
+from acceptance_tests.features.pages import collection_exercise_details
+
+
+@given('the 2017 collection exercise for the BRES survey has been created')
+def bres_2017_exists(_):
+    pass
+
+
+@when('the internal user navigates to the collection exercise details page')
+def internal_user_views_2017_bres_collection_exercise(_):
+    collection_exercise_details.go_to('bres', 'BRES_2017')
```

Comment:
`BRES_2017` needs replaced with the correct period for BRES

Reply:
In the int space this is `221_201712` but I'm not sure if that's what it's supposed to be